curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/passing-values-to-functions-with-arguments.md

```diff
@@ -19,7 +19,7 @@ function testFun(param1, param2) {
 }
 ```
 
-Then we can call `testFun`: `testFun("Hello", "World");` We have passed two arguments, `"Hello"` and `"World"`. Inside the function, `param1` will equal "Hello" and `param2` will equal "World". Note that you could call `testFun` again with different arguments and the parameters would take on the value of the new arguments.
+Then we can call `testFun` like this: `testFun("Hello", "World");`. We have passed two arguments, `"Hello"` and `"World"`. Inside the function, `param1` will equal "Hello" and `param2` will equal "World". Note that you could call `testFun` again with different arguments and the parameters would take on the value of the new arguments.
```

Comment:
@nhcarrigan Since we have been trying to remove single and double quotes inside code blocks, what you think about the following suggestion?
```suggestion
Then we can call `testFun` like this: `testFun("Hello", "World");`. We have passed two strings arguments, `Hello` and `World`. Inside the function, `param1` will equal the string `Hello` and `param2` will equal the string `World`. Note that you could call `testFun` again with different arguments and the parameters would take on the value of the new arguments.
```

Reply:
Yep, that's a good approach.

Reply:
@kelvinsanchez15 Can you commit my suggestion?

Reply:
Committed @RandellDawson 👍